tests/installation/releasenotes_offline.pm

```diff
@@ -21,11 +21,12 @@ sub run {
     assert_screen('release-notes-button');
     send_key('ctrl-shift-alt-x');
     assert_screen('yast-xterm');
-    my $src = check_var('SCC_REGISTER', 'installation') ? "RPM" : "URL";
+    # This is releasenotes_offline so we expect release notes from RPM
+    my $src = "RPM";
     enter_cmd "grep -o \"Got release notes.*\" /var/log/YaST2/y2log";
     assert_screen [qw(got-releasenotes-RPM got-releasenotes-URL)];
     unless (match_has_tag "got-releasenotes-$src") {
-        die "Release notes source does NOT match expectaions or not found in YaST logs, expected source: $src";
+        record_soft_failure("Release notes source does NOT match expectaions or not found in YaST logs, expected source: $src");
```

Comment:
existing typo fixed in accidental PR reintroduced here again: s/expectaions/expectations/

Reply:
In offline scenario we don't soft-fail, we just die if there are not found by rpm, there is not network connection, if the logs would show url would be wrong and we still need to die, you could use `check_screen` for that case, but...

The correct way to do it is to switch tty instead of using needling, but that would mean to change the whole test, so I would recommend you to just check needle rpm and die, we can revisit this in future.